Added post install messages in .opam file

kachida opened the pull request:
added post opam install messages

zapashcanon commented on owi.opam:
```diff
@@ -89,3 +89,9 @@ depexts: [
 ]
+post-messages: [
+  "Thank you for installing owi!"
+  "To fully utilize owi, consider installing a solver such as Z3 or CVC4."
```
```suggestion
  "To fully utilize Owi, consider installing a solver such as Z3."
```

We don't have support for CVC4.

kachida replied:
updated the code.

kachida commented:
> Thanks for the contribution.
> 
> The `owi.opam` file is actually generated by `dune` and will be erased. You should put this in `owi.opam.template` instead.
> 
> I also left some minor suggestions.

Thank you @zapashcanon for guiding me, I've addressed comments, can you please re-review it

zapashcanon commented:
Thanks! I think you are missing a newline at the end of `owi.opam.template`. Also, you should run `dune build owi.opam` to regenerate the file.

kachida commented:
> Thanks! I think you are missing a newline at the end of `owi.opam.template`. Also, you should run `dune build owi.opam` to regenerate the file.

Thanks @zapashcanon , I've updated code, can you please review again

zapashcanon commented:
Thanks!
